frontend/src/entry.client.tsx

```diff
@@ -6,13 +6,25 @@
  */
 
 import { RemixBrowser } from "@remix-run/react";
-import { startTransition, StrictMode } from "react";
+import React, { startTransition, StrictMode } from "react";
 import { hydrateRoot } from "react-dom/client";
 import { Provider } from "react-redux";
+import posthog from "posthog-js";
 import { SocketProvider } from "./context/socket";
 import "./i18n";
 import store from "./store";
 
+function PosthogInit() {
+  React.useEffect(() => {
+    posthog.init("phc_3ESMmY9SgqEAGBB6sMGK5ayYHkeUuknH2vP6FmWH9RA", {
```

Comment:
should probably put this in `config.json`

Reply:
Will do in a follow-up